spec/unit/associations_spec.rb

```diff
@@ -40,7 +40,7 @@
 
   describe Site do
     if Mongoid::Compatibility::Version.mongoid6_or_newer?
-      it { is_expected.to have_many(:users).as_inverse_of(:site).ordered_by(:email.desc).with_counter_cache }
+      it { is_expected.to have_many(:users).as_inverse_of(:site).ordered_by(:email.desc) }
```

Comment:
I don't think I understand this change, care to explain? Thanks.

Reply:
When running specs against Mongoid 7, it complains about the counter_cache option that's not valid. Actually counter_cache should be placed on the belongs_to side of the relationship, doesn't it?

Reply:
@dblock  In 3e80d95 I've fixed counter_cache testing in every mongoid version. Does it look good?